articles/native-platforms/ios-objc/07-authorization.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Authorization
+description: This tutorial will show you how assign roles to your users, and use those claims to authorize or deny a user to perform certain actions in the app.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
```

Comment:
@sebacancinos This does not seem to point to the correct sample. Isn't this the valid one? https://github.com/auth0-samples/auth0-ios-objc-sample/pull/9